scss/components/_banner.scss

```diff
@@ -18,14 +18,17 @@
         background: linear-gradient(180deg, $yellow -20%, rgba(217, 217, 217, 0) 51.04%);
         transform: rotate(-180deg);
     }
-    .container{
+    .banner-wrap{
         max-width: 1200px;
         margin: 0 auto;
         height: 80vh;

```

Comment:
Unfortunately, I have found out that `vh` is not supported, at least in the authoring environment. Would there be a `%` height we could use that would be close enough?

Reply:
Changed to 50em... as % wont work